sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs

```diff
@@ -12,28 +12,49 @@
 using System.Threading.Tasks;
 using System.Threading;
 using Microsoft.Extensions.Logging;
+using System.Runtime.CompilerServices;
 
 namespace WebAssembly.Net.Debugging {
 	internal class BreakpointRequest {
+		public string Id { get; private set; }
 		public string Assembly { get; private set; }
 		public string File { get; private set; }
 		public int Line { get; private set; }
 		public int Column { get; private set; }
 
+		JObject request;
+
+		public bool IsResolved => Assembly != null;
+		public List<Breakpoint> Locations { get; } = new List<Breakpoint> ();
+
 		public override string ToString () {
 			return $"BreakpointRequest Assembly: {Assembly} File: {File} Line: {Line} Column: {Column}";
 		}
 
-		public static BreakpointRequest Parse (JObject args, DebugStore store)
+		public object ToObject ()
```

Comment:
maybe `s,ToObject,ToBreakpoinByUrlResponse,`. The response object for `setBreakpoint` is different from this. The interface for chrome devtools names this as `export interface SetBreakpointByUrlResponse {`.

Reply:
The plan eventually is to serialize these with the json serializer properly but for now renamed to AsSetBreakpointByUrlResponse